frontend/public/components/utils/webhooks.tsx

```diff
@@ -90,7 +90,7 @@ export const WebhookTriggers: React.FC<WebhookTriggersProps> = (props) => {
       setWebhookSecrets(_.compact(secrets));
       setLoaded(true);
     });
-  }, [isLoaded, canGetSecret]);
+  }, [isLoaded, canGetSecret, secretNames, namespace]);
```

Comment:
This change isn't right. We should remove from this PR. It's already fixed under #3929

Reply:
The linter added this one. I'll rebase and remove it, but I was getting test failures without it.